logical/translate_response.go

```diff
@@ -92,6 +94,7 @@ type HTTPResponse struct {
 	Warnings      []string               `json:"warnings"`
 	Headers       map[string][]string    `json:"-"`
 	Auth          *HTTPAuth              `json:"auth"`
+	Namespace     string                 `json:"namespace,omitempty"`
```

Comment:
I don't think we need to add this to the top level response object here

Reply:
The fields in `logical.Response.Secret` gets translated into the top-level field of `HTTPResponse`, see https://github.com/hashicorp/vault/pull/6212/files#diff-149879c600c98fcad95ce83cf71dea10R25 for `Response.Secret.Namespace` as well as other Secret related values.